src/components/dar_dataset_table/DarDatasetTable.jsx

```diff
@@ -170,6 +163,7 @@ export const DarDatasetTable = (props) => {
       );
       setBuckets(dataAccessBuckets);
       setTableSize(dataAccessBuckets.length);
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Could you instead rename `error` to `_error`? Looking at https://github.com/DataBiosphere/duos-ui/blob/develop/eslint.config.js#L57 it appears that using a name that starts with an underscore should fix this eslint warning.

Reply:
Great idea, will give that a shot